packages/cli/templates/default-backend-module/README.md.hbs

```diff
@@ -1,5 +1,5 @@
 # {{name}}
 
-The {{moduleId}} backend module for the {{pluginId}} plugin.
+The {{moduleId}} backend module for the {{id}} plugin.
```

Comment:
I think it's best to keep `pluginId` too, much more explicit and feels safer. It's especially what we stick to in the backend system with `pluginId` / `moduleId`. I think this is even more important in this case for modules where you have both the `moduleId` and `pluginId`.

Reply:
Yep there's certainly a lot of built-in cleanup work from the variation in the existing templates in this PR. I'm just aiming to help guide that 😁